### DISCUSSION TITLE
Issue accessing common CLI commands

### DISCUSSION THREAD

**Discussion by jmasselink** [ohmyzsh/ohmyzsh#12573] [Help] [answered]
I am having an issue accessing some of my commonly used CLI tools like psql (PostgreSQL) and GDAL/OGR. Conda is working.
Here is my .zsh profile [ohmyzsh.zshrc.txt](https://github.com/user-attachments/files/16337280/ohmyzsh.zshrc.txt) and PATH variables (one per line for clarity). Any suggestions? Thanks!

╭─mac@mac /Applications/QGIS-LTR.app/Contents/MacOS/bin  
╰─➤  echo $PATH
/Applications/QGIS-LTR.app/Contents/MacOS/bin:
/opt/miniconda3/condabin:
/usr/local/bin:
/usr/bin:
/bin:
/usr/sbin:
/sbin:
/Library/Apple/usr/bin:
/Applications/Postgres.app/Contents/Versions/latest/bin

**Comment by mcornella** [comment]
Essentially what you want is for the folders that have these commands to be in `$PATH`. Have you checked where those commands are located?

**Reply by jmasselink** [reply]
Thanks! I've gotten the GDAL/OGR configured well now by adding this:
```
# Custom PATH additions
export PATH="/Applications/QGIS-LTR.app/Contents/MacOS/bin:$PATH"
```
However, I'm unsure of how to proceed with specifying the location of my dockerized Postgres container.
```
# Dockerized Postgres container path
export PATH="$HOME/var/run/postgresql:$PATH"
```

**Reply by mcornella** [reply, marked as answer]
Commands within a container don't live in the same filesystem as the host machine. You'll have to `docker exec` within the container and run the command in there, or otherwise set this up so that a `psql` command exists within the host filesystem.
